src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier.js

```diff
@@ -42,12 +42,13 @@ const includeClassifier = ({ href, pathname }) => {
     return { includeType, classification: 'vj-canonical' };
   }
 
-  if (includeType === 'vj' && ampSupported(href)) {
+  if (includeType === 'vj' && hasAmpVersion(href)) {
     return { includeType, classification: 'vj-supports-amp' };
   }
 
-  if (includeType === 'vj' && !ampSupported(href)) {
-    return { includeType, classification: 'not-supported' };
+  if (includeType === 'vj' && !hasAmpVersion(href)) {
+    // Since fallbacks for VJs are supported we must change the name of the classification to prevent returning nothing
```

Comment:
This comment confuses me. 😅 
- "Since fallbacks for VJs are supported" - wouldn't we always 'support' a fallback?
- What does "prevent returning nothing" mean?

I wonder if this comment is needed at all, since it's clear what this control block is doing.